app/javascript/admin/controllers/consumer_app_controller.js

```diff
@@ -0,0 +1,17 @@
+import { Controller } from 'stimulus';
+
+export default class ConsumerAppController extends Controller {
```

Comment:
🙌🏼 on giving this a name. Seems like a lot of the Stimulus docs recommend exporting an anonymous class, but this generally reads better in stack traces and profiling/debugging reports. :100:

Reply:
I can't take credit for this because I took a look at other Stimulus controllers we currently have in the codebase. I will take credit on keeping a close eye on the good practices already in place to maintain consistency though 😆